### PR TITLE
Clean up rumble commands

### PR DESCRIPTION
**Pull request by suryatho**

### CODE REVIEW

File: src/main/java/org/littletonrobotics/frc2024/RobotContainer.java [hunk 1]
```diff
@@ -290,41 +289,31 @@ public RobotContainer() {
...
-                Commands.waitSeconds(0.1),
-                controllerRumbleCommandFactory.apply(0.2),
-                Commands.waitSeconds(0.1),
-                controllerRumbleCommandFactory.apply(0.2)));
+            controllerRumbleCommand(true, true, true)
+                .withTimeout(0.2)
+                .andThen(
+                    Commands.waitSeconds(0.1)
+                        .repeatedly()
+                        .withTimeout(0.9) // Rumble three times
```
Comment (jwbonner) [inline src/main/java/org/littletonrobotics/frc2024/RobotContainer.java:314]: pls fix thx

**Review by jwbonner — CHANGES_REQUESTED** [review]

File: src/main/java/org/littletonrobotics/frc2024/RobotContainer.java (hunk 1 above)
Comment (suryatho) [reply src/main/java/org/littletonrobotics/frc2024/RobotContainer.java:314]: 👍

File: src/main/java/org/littletonrobotics/frc2024/RobotContainer.java [hunk 2]
```diff
@@ -763,6 +734,25 @@ private void configureButtonBindings() {
                 .ignoringDisable(true));
   }
 
+  /** Creates a controller rumble command with specified rumble and controllers */
+  private Command controllerRumbleCommand(boolean useRightRumble, boolean rumbleOperator) {
```
Comment (jwbonner) [inline src/main/java/org/littletonrobotics/frc2024/RobotContainer.java:738]: Every instance where this method is called passes the same arguments...

**Review by jwbonner — CHANGES_REQUESTED** [review]

File: src/main/java/org/littletonrobotics/frc2024/RobotContainer.java [hunk 3]
```diff
@@ -506,6 +494,7 @@ private void configureButtonBindings() {
                     rollers.setGoalCommand(Rollers.Goal.FEED_TO_SHOOTER),
                     superstructureAimCommand.get(),
                     flywheels.shootCommand()));
+    driver.a().and(readyToShoot).whileTrue(controllerRumbleCommand());
```
Comment (jwbonner) [inline src/main/java/org/littletonrobotics/frc2024/RobotContainer.java:497]: This line seems to appear twice...

**Review by jwbonner — CHANGES_REQUESTED** [review]

File: src/main/java/org/littletonrobotics/frc2024/RobotContainer.java (hunk 3 above)
Comment (suryatho) [reply src/main/java/org/littletonrobotics/frc2024/RobotContainer.java]: fixed

**Review by jwbonner — APPROVED** [review]

**Pull request merged**